5_databases/week2/homework part1.sql

```diff
@@ -0,0 +1,28 @@
+--Add a task with these attributes: title, description, created, updated, due_date, status_id, user_id
+INSERT INTO task (title, description, created, updated, due_date, status_id, user_id)
+ VALUES ("Learn web development", "HTML, CSS, JS, React,etc", "2023-11-15 11:14:00", "2023-11-15 11:14:00", "2024-04-30 11:14:00" ,"2", "1");
+
+--Change the title of a task
+UPDATE task
+SET title = "Completed web development"
+WHERE title = "Learn web development";
```

Comment:
Correct, but better to do with a primary key (id) so you dont edit more than one